src/data.js

```diff
@@ -0,0 +1,46 @@
+
+
+export const SuggestedProducts = [
```

Comment:
Suggested Products will require a logic, that contains data based on user's behaviour from database results. For now we use static data, but in future please consider to implemening dinamic functionalities in such cases.